.github/workflows/ci_static-analysis.yaml

```diff
@@ -24,17 +23,18 @@ jobs:
           - flake8
           - pylint
           - flake8-tests
+          - flake8-examples
```

Comment:
should this also add pylint-examples?

Reply:
I initially added pylint, and it produces a lot of errors. Do we want to include the formatting changes in this PR? If yes, I'll go ahead and get those changes in!

Reply:
What sort of errors? If it's something like doc errors that don't make sense for examples, we could just exclude those in the pylint-examples target. If it's a bunch of other random things that we don't necessarily want to update our examples with, then I'd also be fine with leaving it out.

Reply:
It's mostly doc errors, no pydoc strings for methods, or unused variables (some of which customers can use to print to the console to see)